lib/OrdinaryDiffEqRosenbrock/src/rosenbrock_perform_step.jl

```diff
@@ -1253,55 +1235,55 @@ end
     du = f(uprev, p, t)
     integrator.stats.nf += 1
 
-    linsolve_tmp = du + dtd1 * dT
```

Comment:
this entire thing should be a `for` loop that loops over all the steps (so that it will work for arbitrary sized tableau)